framework/src/ble_manager/ble_manager_msghandler.c

```diff
@@ -86,12 +86,11 @@ int blemgr_post_message(blemgr_msg_s *msg)
 		return -1;
 	}
 
-	res = sem_wait(hmsg.signal);
-	if (res < 0) {
-		sem_destroy(hmsg.signal);
-		BLE_ERR;
-		return -2;
+	while(sem_wait(hmsg.signal) < 0)
+	{
```

Comment:
Let's follow TizenRT coding rules.
```
while (sem_wait(...) < 0) {
```
1. add a space after while
2. locate `{` at end of the line